lib/kino/subscription_manager.ex

```diff
@@ -92,7 +113,8 @@ defmodule Kino.SubscriptionManager do
     {subscribers, state} = pop_in(state.topic_with_subscribers[topic])
 
     state =
-      Enum.reduce(subscribers || [], state, fn {pid, _tag}, state ->
+      Enum.reduce(subscribers || [], state, fn {pid, tag}, state ->
+        send(pid, {:__topic_cleared__, topic, tag})
```

Comment:
Isn't this going to send it for all subscribers? Even the ones that are not streams? 🤔

Reply:
Yeah, alternatively we can subscribe with a special tag like `{:__stream__, ref}` and send message only for those tags, wdyt?

Reply:
Either way is fine. We can send it to everyone, but if we do, then we need to document it. Weren't we planning on having join and leave events anyway?

Reply:
The join/leave events concern the user, not the widget disappearing. I think it makes more sense to message the stream only, because otherwise we may spam the subscribing process and there's likely nothing it can do, just waits for cell reevaluation.

Reply:
But for keyboard, we want to know when the keyboard is deactivated and the user leaves the page, right? In any case, keeping this only for streams for now is a great starting point.

Reply:
For keyboard the user decides if they want the `:status` events, if so they receive event on activation/deactivation. It's different from keyboard being removed on cell reevaluation though.

Reply:
Done!